clang-tools-extra/clangd/index/remote/marshalling/Marshalling.cpp

```diff
@@ -189,6 +189,10 @@ llvm::Expected<clangd::Ref> Marshaller::fromProtobuf(const Ref &Message) {
     return Location.takeError();
   Result.Location = *Location;
   Result.Kind = static_cast<RefKind>(Message.kind());
+  auto ContainerID = SymbolID::fromStr(Message.container());
+  if (!ContainerID)
+    return ContainerID.takeError();
```

Comment:
this shouldn't be a failure, as an old server might provide a ref response without a container, but the clients should still keep working.